ibis/omniscidb/client.py

```diff
@@ -479,6 +433,48 @@ def _alter_table_helper(self, f, **alterations):
             results.append(result)
         return results
 
+    def add_column(
+        self, cols_with_types, nullables=None, defaults=None, extras=None
```

Comment:
@YarShev I think theses parameters would be a little bit confusing to the user... 

also maybe you should change `extra` to `encoding` to be more specific.

so I have 2 ideas about this implementation:

1. maybe would be good to just create 1 field at once (also if the performance is not too good) .
2. if you want to keep it for multiple columns
2.a.  maybe a better option would be use schema to represent cols, types and nullables ... but you will still need  to add parameters for defaults and encoding
2.b or use a list of dictionaries when each key would be the name of the desired parameter
`cli.add_column([{'name': 'a', 'type': 'int8', 'nullable': False, 'default': 1}])`

I think we can start with option 1 and start an issue to discuss about the best way to implement `add_columns` method.

what do you think about this?
@jreback what is your thought about this?

Reply:
I think the option is to add multiple columns at once is very nice both for convenience (not call multiple times the method for adding one column) and for performance purposes. Considering your options suggested I think the best opt is 2.b. So, I am going to start with this option. Will see, how will it look.

Reply:
I think the option is to add multiple columns at once is very nice both for convenience (not call multiple times the method for adding one column) and for performance purposes. Considering your options suggested I think a more appropriate opt is 2.b. But then we make users explicitly specify every attribute of column (name, type, nullable, default, encoding) as dictionary's keys. Need to reach a trade-off. What do you think @xmnlab , @jreback ?

Reply:
At the moment I stick current implementation.

Reply:
I think the option is to add multiple columns at once is very nice both for convenience (not call multiple times the method for adding one column) and for performance purposes. Considering your options suggested I think a more appropriate opt is 2.b. But then we make users explicitly specify every attribute of column (name, type, nullable, default, encoding) as dictionary's keys what isn't necessary sometimes. Need to reach a trade-off. What do you think @xmnlab , @jreback ?

Reply:
@jreback, just a friendly reminder to review this PR.